components/org.wso2.carbon.business.rules.web/src/components/pages/landingpage/elements/BusinessRule.jsx

```diff
@@ -185,6 +234,9 @@ export default class BusinessRule extends Component {
         );
     }
 
+
```

Comment:
remove these unnecessary line spaces. plz check the other places also

Reply:
I will check and do the changes